artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -283,6 +283,12 @@
    @Option(name = "--staticCluster", description = "Cluster node connectors list, separated by comma: Example \"tcp://server:61616,tcp://server2:61616,tcp://server3:61616\"")
    String staticNode;
 
+   @Option(name = "--support-advisory", description = "Set supportAdvisory=true for the main 'artemis' acceptor on OPENWIRE protocol (Default: false)")
```

Comment:
On Boolean values, the defaults are always false.

instead, please change the text as... 


description = "If you use --support-advisory, the generated configuration will allow advisories for the openwire protocol"